exercises/collatz-conjecture/src/test/java/CollatzCalculatorTest.java

```diff
@@ -23,43 +23,43 @@ public void setUp() {
 
     @Test
     public void testZeroStepsRequiredWhenStartingFrom1() {
-        assertEquals(0, collatzCalculator.computeStepCount(1));
+//        assertEquals(0, collatzCalculator.computeStepCount(1));
```

Comment:
As per the discussion in PR #683, we decided not to comment out tests after all as this might be confusing to people solving the exercise. The [POLICIES doc](https://github.com/exercism/java/blob/master/POLICIES.md) has been updated to reflect this. So all you have to do is to remove the `CollatzCalculator.java` file completely and replace it with a `.keep` file (the `.keep` file basically tells git not to ignore that directory even though it's empty). Sorry for the confusion! :)